lib/Installation/ModuleRegistration/ModuleRegCodePage.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The module provides interface to act with page that ask for module extra registration code
+#
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package Installation::ModuleRegistration::ModuleRegCodePage;
+use parent 'Installation::Navigation::NavigationBase';
+use strict;
+use warnings;
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {
+        app => $args->{app}
+    }, $class;
+    return $self->init();
+}
+
+sub init {
+    my ($self) = @_;
+    $self->SUPER::init();
+    $self->{tb_we_code} = $self->{app}->textbox({id => '"sle-we"'});
```

Comment:
Is that possible we match generic id instead of sle-we?  Otherwise this code can ONLY handle sle-we code page, or we create module name for ModuleRegWeCodePage?

Reply:
we can postpone this as well, I would have preferred to use the label instead of the tb, but it is ok IMO for now. In the future we will fix it with more regcode needed for ltss.

Reply:
Yes, I understand that we follow the pattern "1 module -> 1 page" , so I'm ok with renaming to be more specific

Reply:
it is the same screen, just a different label, will be more practical to include all the modules in this page that need extra code (in the future) so I would not rename it, wdyt?

Reply:
+1, make sense for me (and conforms to existing code)